website/wasm-compiler/src/server.ts

```diff
@@ -0,0 +1,60 @@
+import express, { Request, Response, Express, json } from 'express';
+import multer from 'multer';
+import { generateId, unpackZip } from './util';
+import { DBService } from './db';
+import { CompilerService } from './compiler';
+import config from './configuration';
+import { resolve } from 'path';
+
+const upload = multer();
+
+export class Server {
+  dbService: DBService;
+  compiler: CompilerService;
+  app: Express;
+
+  constructor(dbService: DBService, compiler: CompilerService) {
+    this.dbService = dbService;
+    this.compiler = compiler;
+    this.app = express();
+    this.app.use(json());
```

Comment:
it seems to need add default error handler for not caught errors, otherwise application crushes because of this

Reply:
There are only 2 endpoints in the service, so I guess try / catch is more than enough. And this service won't be used much